doc/adrs/191_add_diagram_elements_appearance_management.adoc

```diff
@@ -0,0 +1,27 @@
+= ADR-191 - Add the capability of customizing the appearance of diagram elements
+
+== Context
+
+The appearance of diagram elements is not customizable outside of what is defined in the specifier.
+
+== Decision
+
+End-users will be able to customize the appearance of diagram elements through the palette, these customizations will be persisted but can be erased to fall back to the specified appearance.
+
+== Backend
+
+A new field in nodes and edges will store their customized appearance, the field may be null when there is no customization.
```

Comment:
Why should we have two identical data structure? Why not have only one and keep the list of overriden properties. We could keep the non-nullability and remove the need to maintain two copies of the same data structure.

Reply:
There was no concept of independent style properties but yes we could add it instead of the data structur mirror.